### ISSUE TITLE
Launching `bun repl` crashes on Mac

### ISSUE THREAD

**Issue by prantlf** [oven-sh/bun#12081]
### How can we reproduce the crash?

Just launch `bun repl` with no other command-line arguments.

MacOS: Version 14.5 (23F79)
Hardware: MacBook 2018 2,6 GHz 6-Core Intel Core i7

Launching `bun repl` crashed also at #10914 (Windows), #11015 (Windows) and #11359 (Linux).

### Relevant log output

```shell
❯ bun repl
============================================================
Bun v1.1.16 (bf7b327f) macOS x64
Args: "bun" "bun-repl"
Elapsed: 8ms | User: 6ms | Sys: 12ms
RSS: 2.82MB | Peak: 2.82MB | Commit: 1.07GB | Faults: 378

panic(main thread): Bus error at address 0x107482992
oh no: Bun has crashed. This indicates a bug in Bun, not your code.

To send a redacted crash report to Bun's team,
please file a GitHub issue using the link below:

 https://bun.report/1.1.16/mB1bf7b327AA__q+3/Di5whF4hyB2wqB__A4Ck50gpH

zsh: illegal hardware instruction  bun repl
```

### Stack Trace (bun.report)

Bun v1.1.16 ([`bf7b327`](<https://github.com/oven-sh/bun/tree/bf7b327f68568104a5e40c1944b7b255b02f0745>)) on macos x86_64 [BunxCommand]

Bus error at address 0x10DD2D992

- *2 unknown/js code*
- `src.install.install.PackageManager.UpdateRequest.parse`
- `src.cli.bunx_command.BunxCommand.exec`
- `src.cli.Cli.start`
- `main`
- *2 unknown/js code*

**Comment by Jarred-Sumner** [comment]
Duplicate of https://github.com/oven-sh/bun/issues/12070

**Comment by Jarred-Sumner** [comment]
This was fixed in #12071 

To upgrade, please run `bun upgrade --canary`

**Comment by prantlf** [comment]
Yes, it works, thank you for the quick answer!
